_posts/2022-12-28-courseware.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+author: Jehangir Amjad
```

Comment:
This should include you + Crystal Wang as well

Reply:
Added more authors!